src/main.jsx

```diff
@@ -1,10 +1,13 @@
-import React from "react";
+import React, { StrictMode } from "react";
```

Comment:
It's better to remove the React import since it's not being used anywhere.

Reply:
Ahhh yeah! Thanks for the catch